python/lsst/scarlet/lite/initialization.py

```diff
@@ -271,6 +271,10 @@ class FactorizedInitialization:
     convolved:
         Deprecated. This is now calculated in __init__, but the
         old API is supported until v29.0.
+    is_symmetric:
+        Whether or not the source is symmetric.
+        This is used to determine if the morphology should be
+        forced to be symmetric.
     """
```

Comment:
Missing doc for is_connected. bg_thresh and initial_bg_thresh are reversed in order in the docstring vs args, FWIW.